client/components/plans/plan-list/index.jsx

```diff
@@ -46,12 +46,12 @@ module.exports = React.createClass( {
 			);
 		}
 
-		if ( this.props.sites ) {
-			site = this.props.sites.getSelectedSite();
-			showJetpackPlans = site && site.jetpack;
-		}
-
 		if ( ! this.props.isInSignup ) {
+			if ( this.props.sites ) {
```

Comment:
This change improves clarity in children components because we don't want `site` to be available for them during signup.